### PR TITLE
Move to `1.13.0-SNAPSHOT`

### PR DESCRIPTION
**Pull request by Fokko**
Now 1.12.0 has been released 🥳 

## What is the purpose of the change

*(For example: This pull request improves file read performance by buffering data, fixing AVRO-XXXX.)*

## Verifying this change

*(Please pick one of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
- *Extended interop tests to verify consistent valid schema names between SDKs*
- *Added test that validates that Java throws an AvroRuntimeException on invalid binary data*
- *Manually verified the change by building the website and checking the new redirect*

## Documentation

- Does this pull request introduce a new feature? (yes / no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by martin-g** [comment]
Most of the other projects use the contents of .../share/VERSION.txt but only the JS SDK does not! Let's also touch lang/js/package.json

**Comment by martin-g** [comment]
The Java build fails with the same symptoms as the problems with the Docker based testing for the 1.12.0 release - https://github.com/apache/avro/actions/runs/10302739342/job/28517089604?pr=3076
The Maven archetype does not see the 1.13.0-SNAPSHOT artifacts in the reactoer build.

**Comment by Fokko** [comment]
@martin-g The `package.json` is already updated: https://github.com/apache/avro/pull/3076/files#diff-4390897d44bcb4480050fea93917b5f82e63877c33832bfd3262528ab4f761b6R3

**Comment by Fokko** [comment]
It looks like there are nightly builds available: 

image

But this looks like a chicken and egg situation :/

File: .github/workflows/test-lang-java.yml [hunk 1]
```diff
@@ -63,15 +63,19 @@ jobs:
         with:
           maven-version: 3.9.6
 
+      - name: 'Install Java Avro'
+        working-directory: .
```
Comment (martin-g) [inline .github/workflows/test-lang-java.yml:67]: Is this needed ?

Comment (Fokko) [reply .github/workflows/test-lang-java.yml:67]: We want to install it from top-level:
```suggestion
        working-directory: /runner/_work/avro/avro
```

File: .github/workflows/test-lang-java-ARM.yml [hunk 2]
```diff
@@ -65,6 +65,9 @@ jobs:
         with:
           maven-version: 3.9.6
 
+      - name: 'Install Java Avro'
```
Comment (Fokko) [inline .github/workflows/test-lang-java-ARM.yml:68]:
```suggestion
      - name: 'Install Java Avro'
        working-directory: /runner/_work/avro/avro
```

**Review by Fokko — COMMENTED** [review]
Just giving this a try, when it works I'll come up with a cleaner solution

**Review by martin-g — APPROVED** [review]

**Pull request merged**